dex/seeds/trades/maverick_zksync_base_trades_seed.csv

```diff
@@ -1,2 +1,3 @@
 blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
 zksync,maverick,1,2023-10-23,0xbe2700aac1174728513863a4f42d37aa9ac7a7d73389fa00a5abcfbd3570d5a6,4,0x2039bb4116B4EFc145Ec4f0e2eA75012D6C0f181,0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4,17114057,1723727835619188444,1723200
+zksync,maverick,2,2024-07-24,0xceb5de2540e7df1ba059a6da0c8a81632736725c54a0c39fb1e62b216cadbad2,88,0x5aea5775959fbc2557cc8789bc1bf90a239d9a91,0x5a7d6b2f92c77fad6ccabd7ee0624e64907eaf3e,39925150,25855263746109189,505494599746163070374
```

Comment:
Please verify this seed record.
https://dune.com/queries/3941353

Reply:
It seems the zksync factory contract was misnamed (I submitted a renaming request to `V2Factory` a few days ago, but it is still in the queue.  

Using the existing name, I can run this in dune and i get the expected result:

```
SELECT
  t.evt_block_time AS block_time,
  t.evt_block_number AS block_number,
  t.recipient AS taker,
  TRY_CAST(NULL AS VARBINARY) AS maker,
  t.amountOut AS token_bought_amount_raw,
  t.amountIn AS token_sold_amount_raw,
  CASE
    WHEN TRY_CAST(JSON_EXTRACT_SCALAR(t.params, '$.tokenAIn') AS BOOLEAN) = TRUE
    THEN f.tokenB
    ELSE f.tokenA
  END AS token_bought_address,
  CASE
    WHEN TRY_CAST(JSON_EXTRACT_SCALAR(t.params, '$.tokenAIn') AS BOOLEAN) = TRUE
    THEN f.tokenA
    ELSE f.tokenB
  END AS token_sold_address,
  t.contract_address AS project_contract_address,
  t.evt_tx_hash AS tx_hash,
  t.evt_index
FROM maverick_v2_zksync.V2Pool_evt_PoolSwap AS t
INNER JOIN maverick_v2_zksync.MaverickV2Factory_evt_PoolCreated AS f
  ON t.contract_address = f.poolAddress
WHERE
  t.evt_tx_hash = 0xceb5de2540e7df1ba059a6da0c8a81632736725c54a0c39fb1e62b216cadbad2
  ```
  
  But the CI can't seem to find `maverick_v2_zksync.MaverickV2Factory_evt_PoolCreated`.  any ideas, @Hosuke ?

Reply:
We may need to add the new source table into the corresponding`sources.yml`.